doc/reference/tooling/tcm/tcm_access_control.rst

```diff
@@ -0,0 +1,383 @@
+
+Passwords expire automatically after the expiration period defined in the :ref:`password policy <tcm_access_control_password_policy>`.
+When a user logs in to |tcm| with an expired password, the only action available to
+them is password change. All other |tcm| functions and objects are unavailable until
+the new password is set.
+
+Administrators can also set users' password to expired manually.
+To set a user's password to expired, click **Expire** in the **Actions**
+menu of the corresponding **Secrets** table row.
+
+.. important::
+
+    Password expiration can't be reverted.
+
+Blocking passwords
+~~~~~~~~~~~~~~~~~~
+
+To forbid users' access to |tcm|, administrators can temporarily block their
+passwords. A blocked password can't be used to log into |tcm| until it's
+unblocked manually or the blocking period expires.
+
+To block a user's password, click **Block** in the **Actions** menu of the corresponding
+**Secrets** table row. Then provide a blocking reason and enter the blocking period.
+
+To unblock a blocked password, click **Unblock** in the **Actions** menu of the corresponding
+**Secrets** table row.
+
+.. _tcm_access_control_password_policy:
+
+Password policy
+~~~~~~~~~~~~~~~
+
+Password policy helps improve security and comply with security requirements that
+can apply to your organization.
+
+You can edit the |tcm| password policy on the **Password policy** page.
+There are the following password policy settings:
+
+-   **Minimal password length**.
+-   **Do not use last N passwords**.
+-   **Password expiration in days**. After this number of days, the user loses access
+    to any objects and functions except password change. After they change the password,
+    all the access rights are returned.
+-   **Password expiration warning in days**. After this number of days, the user
+    sees a warning that they must change their password.
+-   **Block after N login attempts**.
+-   **User lockout time in seconds**.
+-   **Password must include**. Characters and symbols that must be present in passwords:
+
+    -   **Lowercase characters (a-z)**
+    -   **Uppercase characters (A-Z)**
+    -   **Digits (0-9)**
+    -   **Symbols (such as !@#$%^&\*()_+№"':,.;=][{}`?>/.)**
+
+.. _tcm_access_control_sessions:
+
+Sessions
+--------
+
+Administrators can view and revoke user sessions in |tcm|. All active sessions
+are listed on the **Sessions** page. To revoke a session, click **Revoke** in the
+**Actions** menu of the corresponding table row.
+
+To revoke all sessions of a user, go to **Users** and click **Revoke all sessions**
+in the **Actions** menu of the corresponding table row.
+
+.. _tcm_access_control_permissions_ref:
+
+Permissions reference
+---------------------
+
+.. _tcm_access_control_permissions_admin:
+
+Administrative permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following administrative permissions are available in |tcm|:
+
+..  list-table::
+    :widths: 30 70
+    :header-rows: 1
+
+    *   -   Permission
```

Comment:
my bad, 

user.password.change is actual for access to change own password

Reply:
Restored